### PR TITLE
Update JuMP example in README

### PR DESCRIPTION
**Pull request by blegat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,12 +39,13 @@ $$
 You can solve an JuMP model `model` by using [NLPModelsJuMP.jl](https://github.com/JuliaSmoothOptimizers/NLPModelsJuMP.jl) to convert it.
 ```julia
 using JuMP, NLPModelsJuMP, Percival
-model = Model()
+model = Model(NLPModelsJuMP.Optimizer)
```
Comment (blegat) [inline README.md:42]: This is possible now thanks to https://github.com/JuliaSmoothOptimizers/NLPModelsJuMP.jl/pull/157

File: README.md [hunk 2]
```diff
@@ -39,12 +39,13 @@ $$
...
+set_attribute(model, "solver", Percival.PercivalSolver)
 @variable(model, x[i=1:2], start = [-1.2; 1.0][i])
-@NLobjective(model, Min, (x[1] - 1)^2 + 100 * (x[2] - x[1]^2)^2)
-@NLconstraint(model, x[1]^2 + x[2]^2 == 1)
-nlp = MathOptNLPModel(model) # thin wrapper converting JuMP Model as NLPModel
-output = percival(nlp, verbose = 1)
+@objective(model, Min, (x[1] - 1)^2 + 100 * (x[2] - x[1]^2)^2)
```
Comment (blegat) [inline README.md:45]: We can use the new syntax thanks to https://github.com/JuliaSmoothOptimizers/NLPModelsJuMP.jl/pull/171

**Comment by blegat** [comment]
The failure does not seem related

**Review by tmigot — APPROVED** [review]
Looks good to me. Thank you for the PR

**Pull request merged**